src/node_crypto.cc

```diff
@@ -5421,11 +5421,15 @@ void PBKDF2(const FunctionCallbackInfo<Value>& args) {
 // Only instantiate within a valid HandleScope.
 class RandomBytesRequest : public AsyncWrap {
  public:
-  RandomBytesRequest(Environment* env, Local<Object> object, size_t size)
+  RandomBytesRequest(Environment* env,
+                     Local<Object> object,
+                     size_t size,
+                     size_t offset)
       : AsyncWrap(env, object, AsyncWrap::PROVIDER_CRYPTO),
         error_(0),
         size_(size),
-        data_(node::Malloc(size)) {
+        data_(node::Malloc(size)),
```

Comment:
so, if I just pass in `req->data()` without actually copying the memory, it aborts. I was able to switch to using memcpy though.